azuredevops/internal/service/build/resource_resource_authorization.go

```diff
@@ -120,55 +154,64 @@ func resourceResourceAuthorizationDelete(d *schema.ResourceData, m interface{})
 	// because the resource to delete might have had this parameter set to true, we overwrite it
 	authorizedResource.Authorized = converter.Bool(false)
 
-	err = sendAuthorizedResourceToAPI(clients, authorizedResource, projectID)
+	err = sendAuthorizedResourceToAPI(clients, authorizedResource, projectID, definitionID)
```

Comment:
`expandAuthorizedResource` will always return error as`nil`. `if` statement is meaningless here

Reply:
No, the function returns a potential error
![image](https://user-images.githubusercontent.com/14177833/87674398-3bb4e280-c776-11ea-8eac-0e13f46c85b1.png)

Reply:
function `expandAuthorizedResource` not `sendAuthorizedResourceToAPI` , at line 160. I comment on the wrong code line :D